### ISSUE TITLE
No return type for function getCookieConsentValue

### ISSUE THREAD

**Issue by gitanjaliGiri** [Mastermindzh/react-cookie-consent#106]
I am getting the below error when trying to build my solution with the "react-cookie-consent" package

node_modules/react-cookie-consent/build/index.d.ts(52,17): error TS7010: 'getCookieConsentValue', which lacks return-type annotation, implicitly has an 'any' return type.

Its a blocker for us. Kindly help

**Comment by Mastermindzh** [comment]
I can add the return type tonight :)

You can either wait a bit or add a typescript ignore next line where you use getCookieConsentValue :D

**Comment by gitanjaliGiri** [comment]
> I can add the return type tonight :)
> 
> You can either wait a bit or add a typescript ignore next line where you use getCookieConsentValue :D

I would wait for the fix :)

**Comment by Mastermindzh** [comment]
Hey @gitanjaliGiri 

I released a fixed version [6.2.2](https://www.npmjs.com/package/react-cookie-consent/v/6.2.2).
Thank you for bringing this to my attention nd have a lovely day.

**Comment by davorpeic** [comment]
Hello,

still an issue in v7.3
```
node_modules/react-cookie-consent/build/index.d.ts:68:17 - error TS7010: 'resetCookieConsentValue', which lacks return-type annotation, implicitly has an 'any' return type.

68 export function resetCookieConsentValue(name?: string);
```

Fixed it by adding annotation at the end in index.d.ts
`export function resetCookieConsentValue(name?: string): string;`

ps. building with Vite

dp
